conf/csiro_petrichor.config

```diff
@@ -0,0 +1,28 @@
+// CSIRO Petrichor nf-core configuration profile
+params {
+  config_profile_description = 'CSIRO Petrichor HPC profile provided by nf-core/configs'
+  config_profile_contact = 'Mitchell OBrien (@mitchob)'
+  config_profile_url = 'https://confluence.csiro.au/display/SC/CSIRO+SC+Shared+Cluster+-+Petrichor'
+}
+
+// Enable use of Singularity to run containers
+singularity {
+  enabled = true
+  autoMounts = true
+  autoCleanUp = true
+}
+
+// Submit up to XX concurrent jobs 
```

Comment:
Still necessary?

Reply:
Yes, will confirm and come back to this but not essential for now.